core/components/atoms/switch/switch.js

```diff
@@ -29,12 +29,14 @@ class Switch extends React.Component {
   render() {
     let [onLabel, offLabel] = this.props.accessibleLabels
     let elements = [
-      <Checkbox type="checkbox" checked={this.state.on} readOnly id={this.props.id} />
+      <Checkbox key={0} type="checkbox" checked={this.state.on} readOnly id={this.props.id} />
```

Comment:
This is interesting, what does adding a hardcoded key do?

Reply:
It’s there to silence warnings, as proposed by our contributor

Reply:
hard coded though?